scripts/pre-commit.js

```diff
@@ -1,17 +0,0 @@
-'use strict';
```

Comment:
Why was this deleted? See the Generator standard https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/scripts/pre-commit.mjs